README.md

```diff
@@ -83,9 +85,13 @@ Building the documentation copies the full markdown into the `website` folder, r
 
 With the repository cloned to: `$GOPATH/src/github.com/terraform-providers/terraform-provider-fastly`:
 
-* To explicitly build the documentation:  
-`go run scripts/website/parse-templates.go `  
+* To build the documentation:
+`go run scripts/website/parse-templates.go `
 
-* To build and preview the documentation online:  
+* To build and preview the documentation online:
 `make website`
 
+Contributing
```

Comment:
Referred to https://github.com/prometheus/prometheus/blob/master/README.md for inspiration.